### ISSUE TITLE
always return me error code "400 InvalidImageSize" when use computervision api

### ISSUE THREAD

**Issue by RussiaVk** [Azure-Samples/cognitive-services-quickstart-code#98]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Any log messages given by the failure
```
Captions: 
'Bern Collaco et al. posing for a photo' with confidence 0.989732
Status code 400, {"code":"InvalidImageSize","requestId":"b90ca72b-bcd2-4af2-9437-d184fb7e59ce","message":"Image must be at least 50 pixels in width and height"}
null
com.microsoft.azure.cognitiveservices.vision.computervision.models.ComputerVisionErrorException: Status code 400, {"code":"InvalidImageSize","requestId":"b90ca72b-bcd2-4af2-9437-d184fb7e59ce","message":"Image must be at least 50 pixels in width and height"}
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
	at com.microsoft.rest.ServiceResponseBuilder.build(ServiceResponseBuilder.java:122)
	at com.microsoft.azure.AzureResponseBuilder.build(AzureResponseBuilder.java:56)
	at com.microsoft.azure.cognitiveservices.vision.computervision.implementation.ComputerVisionImpl.recognizePrintedTextInStreamDelegate(ComputerVisionImpl.java:833)
	at com.microsoft.azure.cognitiveservices.vision.computervision.implementation.ComputerVisionImpl.access$400(ComputerVisionImpl.java:71)
	at com.microsoft.azure.cognitiveservices.vision.computervision.implementation.ComputerVisionImpl$14.call(ComputerVisionImpl.java:820)
	at com.microsoft.azure.cognitiveservices.vision.computervision.implementation.ComputerVisionImpl$14.call(ComputerVisionImpl.java:816)
	at rx.internal.operators.OnSubscribeMap$MapSubscriber.onNext(OnSubscribeMap.java:69)
	at retrofit2.adapter.rxjava.RxJavaCallAdapterFactory$RequestArbiter.request(RxJavaCallAdapterFactory.java:173)
	at rx.Subscriber.setProducer(Subscriber.java:211)
	at rx.internal.operators.OnSubscribeMap$MapSubscriber.setProducer(OnSubscribeMap.java:102)
	at retrofit2.adapter.rxjava.RxJavaCallAdapterFactory$CallOnSubscribe.call(RxJavaCallAdapterFactory.java:152)
	at retrofit2.adapter.rxjava.RxJavaCallAdapterFactory$CallOnSubscribe.call(RxJavaCallAdapterFactory.java:138)
	at rx.Observable.unsafeSubscribe(Observable.java:10142)
	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:48)
	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:33)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:48)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:30)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:48)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:30)
	at rx.Observable.subscribe(Observable.java:10238)
	at rx.Observable.subscribe(Observable.java:10205)
	at rx.observables.BlockingObservable.blockForSingle(BlockingObservable.java:444)
	at rx.observables.BlockingObservable.single(BlockingObservable.java:341)
	at com.microsoft.azure.cognitiveservices.vision.computervision.implementation.ComputerVisionImpl$ComputerVisionRecognizePrintedTextInStreamParameters.execute(ComputerVisionImpl.java:878)
	at russiavk.module.ocr.api.azure.OCRByLocal(azure.java:79)
	at russiavk.module.ocr.api.azure.main(azure.java:164)
```

### Minimal steps to reproduce
```
package russiavk.module.ocr.api;

import com.microsoft.azure.cognitiveservices.vision.computervision.*;
import com.microsoft.azure.cognitiveservices.vision.computervision.models.*;

import java.io.File;
import java.io.FileInputStream;
import java.io.IOException;
import java.nio.file.Files;

import java.util.ArrayList;
import java.util.List;
import russiavk.module.ocr.base.OCRBase;

public class azure {//extends OCRBase 
	String  SECRET_KEY="**********************";
	String END_POINT="https://regking.cognitiveservices.azure.com/";


//	@Override
//	public OcrClient initAccountData(String... args) { 
//	}

//	@Override
	protected String resultProcessors(Object result) {
		// Print results of local image
		System.out.println();
		System.out.println("Recognizing printed text from a local image with OCR ...");
		System.out.println("\nLanguage: " + ((OcrResult) result).language());
		System.out.printf("Text angle: %1.3f\n", ((OcrResult) result).textAngle());
		System.out.println("Orientation: " + ((OcrResult) result).orientation());

		boolean firstWord = true;
		// Gets entire region of text block
		for (OcrRegion reg : ((OcrResult) result).regions()) {
			// Get one line in the text block
			for (OcrLine line : reg.lines()) {
				for (OcrWord word : line.words()) {
					// get bounding box of first word recognized (just to demo)
					if (firstWord) {
						System.out.println("\nFirst word in first line is \"" + word.text() + "\" with  bounding box: "
								+ word.boundingBox());
						firstWord = false;
						System.out.println();
					}
					System.out.print(word.text() + " ");
				}
				System.out.println();
			}
		}
		return null;

	}

//	@Override
	public String OCRByLocal(String filePath, Object options) {
		ComputerVisionClient compVisClient = ComputerVisionManager.authenticate(SECRET_KEY)
				.withEndpoint(END_POINT);

		// END - Create an authenticated Computer Vision client.

		System.out.println("\nAzure Cognitive Services Computer Vision - Java Quickstart Sample");

		// Analyze local and remote images
		AnalyzeLocalImage(compVisClient);

		// Recognize printed text with OCR for a local and remote (URL) image
		// RecognizeTextOCRLocal(compVisClient);
		try {
			byte[] localImageBytes = Files.readAllBytes(new File(filePath).toPath());

			// Recognize printed text in local image
			OcrResult ocrResultLocal = compVisClient.computerVision().recognizePrintedTextInStream()
					.withDetectOrientation(true).withImage(localImageBytes).withLanguage(OcrLanguages.EN).execute();
			// Print results of local image
			System.out.println();
			System.out.println("Recognizing printed text from a local image with OCR ...");
			System.out.println("\nLanguage: " + ocrResultLocal.language());
			System.out.printf("Text angle: %1.3f\n", ocrResultLocal.textAngle());
			System.out.println("Orientation: " + ocrResultLocal.orientation());

			boolean firstWord = true;
			// Gets entire region of text block
			for (OcrRegion reg : ocrResultLocal.regions()) {
				// Get one line in the text block
				for (OcrLine line : reg.lines()) {
					for (OcrWord word : line.words()) {
						// get bounding box of first word recognized (just to demo)
						if (firstWord) {
							System.out.println("\nFirst word in first line is \"" + word.text()
									+ "\" with  bounding box: " + word.boundingBox());
							firstWord = false;
							System.out.println();
						}
						System.out.print(word.text() + " ");
					}
					System.out.println();
				}
			}
			return resultProcessors(compVisClient);
		} catch (Exception e) {
			System.out.println(e.getMessage());
			e.printStackTrace();
			return null;

		}

	}

	public static void AnalyzeLocalImage(ComputerVisionClient compVisClient) {
		/*
		 * Analyze a local image:
		 *
		 * Set a string variable equal to the path of a local image. The image path
		 * below is a relative path.
		 */
		String pathToLocalImage = "D:/asd.jpg";
		String pathToRemoteImage = "https://github.com/Azure-Samples/cognitive-services-sample-data-files/raw/master/ComputerVision/Images/faces.jpg";

		// This list defines the features to be extracted from the image.
		List<VisualFeatureTypes> featuresToExtractFromLocalImage = new ArrayList<>();
		featuresToExtractFromLocalImage.add(VisualFeatureTypes.DESCRIPTION);
		featuresToExtractFromLocalImage.add(VisualFeatureTypes.CATEGORIES);
		featuresToExtractFromLocalImage.add(VisualFeatureTypes.TAGS);
		featuresToExtractFromLocalImage.add(VisualFeatureTypes.FACES);
		featuresToExtractFromLocalImage.add(VisualFeatureTypes.ADULT);
		featuresToExtractFromLocalImage.add(VisualFeatureTypes.COLOR);
		featuresToExtractFromLocalImage.add(VisualFeatureTypes.IMAGE_TYPE);
		// Need a byte array for analyzing a local image.
		File rawImage = new File(pathToLocalImage);
		byte[] imageByteArray = null;
		try {
			imageByteArray = Files.readAllBytes(rawImage.toPath());
		} catch (IOException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}

		// Call the Computer Vision service and tell it to analyze the loaded image.
//		ImageAnalysis analysis = compVisClient.computerVision().analyzeImageInStream().withImage(imageByteArray)
//				.withVisualFeatures(featuresToExtractFromLocalImage).execute();

		ImageAnalysis analysis = compVisClient.computerVision().analyzeImage().withUrl(pathToRemoteImage)
				.withVisualFeatures(featuresToExtractFromLocalImage).execute();

		// Display image captions and confidence values.
		System.out.println("\nCaptions: ");
		for (ImageCaption caption : analysis.description().captions()) {
			System.out.printf("\'%s\' with confidence %f\n", caption.text(), caption.confidence());
		}
	}

	//
	public static void main(String[] args) {
		try {
			System.out.println(new azure().OCRByLocal(
					""
					null));

		} catch (Exception e) {
			e.printStackTrace();
		}
	}
}
```

### OS and Version?
> Windows 10.

### Versions
>azure-cognitiveservices-computervision1.0.2-beta

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by wiazur** [comment]
Hi @RussiaVk, do you have the image you are trying to use? The errors are saying your image is not big enough. If you attach the image you are using, I can test it. Thanks.

**Comment by RussiaVk** [comment]
> Hi @RussiaVk, do you have the image you are trying to use? The errors are saying your image is not big enough. If you attach the image you are using, I can test it. Thanks.

it give me such error when try any captcha 
![image](https://user-images.githubusercontent.com/43515485/82525246-7aa62d80-9b63-11ea-9c20-f258aaf1883c.png)

**Comment by wiazur** [comment]
Thanks. I did not get an error, but I did get a blank result. I think this is because you used the API call to recognize printed text, and captchas like that are closer to handwritten text. The recognition for handwritten text is in a newer OCR method these days. Try the Batch Read File method and see if that helps. Your image is great than 50px in size, so the image should work OK.

https://docs.microsoft.com/en-us/java/api/com.microsoft.azure.cognitiveservices.vision.computervision.computervision.batchreadfileinstream?view=azure-java-stable#com_microsoft_azure_cognitiveservices_vision_computervision_ComputerVision_batchReadFileInStream_byte____
